ihmc-communication/src/main/java/us/ihmc/communication/crdt/Freezable.java

```diff
@@ -9,7 +9,5 @@ public interface Freezable
 {
    void freeze();
 
-   void unfreeze();
```

Comment:
unfreeze() didn't turn out to be used externally, made it an internal only method